uf3/regression/least_squares.py

```diff
@@ -176,11 +185,17 @@ def set_params(self, **params):
             self.bspline_config = params["bspline_config"]
         if "regularizer" in params:
             self.regularizer = params["regularizer"]
+        if "ridge_map" in params:
+            self.ridge_map = params["ridge_map"]
+        if "curvature_map" in params:
+            self.curvature_map = params["curvature_map"]
```

Comment:
I don't understand why these values are being set again. You already set these values on line 159.